.env

```diff
@@ -19,6 +19,8 @@ REMME_PEERS_LIST=
 REMME_SEEDS_LIST=
 # Consensus algorithm to be used
 REMME_CONSENSUS=
+# Minimum number of peers to start network synchonization
+REMME_MINIMUM_PEERS_CONNECTIVITY=1
```

Comment:
Doesn't **3** is a basic minimum count of peers  ?

Reply:
It is the default value, 1 was set for the testing purposes.